src/PerfView/SymbolReaderHttpHandler.cs

```diff
@@ -725,6 +728,46 @@ private static string GetAuthenticationHeaderScheme(AuthToken authToken)
         }
     }
 
+    internal sealed class BasicHttpAuthHandler : SymbolReaderAuthHandlerBase
+    {
+        /// <summary>
+        /// Prefix to put in front of logging messages.
+        /// </summary>
+        private const string LogPrefix = "BasicAuth: ";
+
+        public BasicHttpAuthHandler(TextWriter log) : base(log, LogPrefix)
+        {
+        }
+
+
+        private static readonly char[] delimiter = { ':' };
+        protected override bool TryGetAuthority(Uri requestUri, out Uri authority)
+        {
+            if (string.IsNullOrEmpty(requestUri.UserInfo) || !requestUri.UserInfo.Contains(":"))
+            {
+                authority = null;
+                return false;
+            }
+            authority = requestUri;
+            return true;
+        }
+
+        protected override Task<AuthToken?> GetAuthTokenAsync(RequestContext context, SymbolReaderHandlerDelegate next, Uri authority,
+            CancellationToken cancellationToken)
+        {
+
+            var strings = authority.UserInfo.Split(delimiter, StringSplitOptions.RemoveEmptyEntries);
+            if (strings.Length < 2)
+            {
+                return base.GetAuthTokenAsync(context, next, authority, cancellationToken);
+            }
+
+            this.WriteStatusLog("auth token for basic auth provided");
+
+            var token = AuthToken.CreateBasicFromUsernameAndPassword(strings[0], strings[1]);
+            return Task.FromResult<AuthToken?>(token);
+        }
+    }
     /// <summary>
```

Comment:
Add missing blank line between closing curly bracket and the start of the doc comment.